tests/fast-integration/kyma-environment-broker/helpers.js

```diff
@@ -92,7 +92,7 @@ async function updateSKR(keb,
 async function ensureOperationSucceeded(keb, kcp, instanceID, operationID, timeout) {
   const res = await wait(
       () => keb.getOperation(instanceID, operationID),
-      (res) => res && res.state && (res.state === 'succeeded' || res.state === 'failed'),
+      (res) => res && res.state && res.state !== 'in progress' && (res.state === 'succeeded' || res.state === 'failed'),
```

Comment:
Can you explain why `res.state !== 'in progress'` is required ? 🤔

Reply:
was a debugging message, removed